src/pybind/mgr/dashboard/module.py

```diff
@@ -895,7 +934,8 @@ def _osd(self, osd_id):
                            }),
                        "")
                 r, outb, outs = result.wait()
-                assert r == 0
+                if r != 0:
+                    return None
```

Comment:
Would make more sense to set histogram to None here rather than returning None.  While removing assertion, should also add a global_instance().log.error so that we're not just dropping the error on the floor.